lib/Onnxifi/Base.cpp

```diff
@@ -91,9 +94,10 @@ onnxStatus Graph::initGraph(const void *onnxModel, size_t onnxModelSize,
   function_ = backendPtr_->getEE().getModule().createFunction("inference");
 
   // TODO: make better error reporting.
-  std::unique_ptr<ONNXIFIModelLoader> loader = TEMP_EXIT_ON_ERR(
-      ONNXIFIModelLoader::parse(onnxModel, onnxModelSize, weightCount,
-                                weightDescriptors, *function_));
+  std::unique_ptr<ONNXIFIModelLoader> loader =
+      TEMP_EXIT_ON_ERR(ONNXIFIModelLoader::parse(
+          onnxModel, onnxModelSize, weightCount, weightDescriptors, *function_,
+          true, backendPtr_->getUseOnnx()));
```

Comment:
Can you add an inline comment with the parameter name corresponding to true?  (I think `true /*loadInputsAsPlaceholders*/`)